src/styles/common/_typography.scss

```diff
@@ -19,6 +19,10 @@ body {
     @include common-typography;
 }
 
+body.is-deja-blue {
```

Comment:
Not going to change this top level class, requires monolith changes. Considering it out of scope right now.